proto/src/main/proto/Rss.proto

```diff
@@ -488,4 +488,26 @@ message CancelDecommissionRequest {
 message CancelDecommissionResponse {
   StatusCode status = 1;
   string retMsg = 2;
-}
\ No newline at end of file
+}
+// ShuffleManager service lives inside of compute-engine's application master, which handles rss shuffle specific logic
+// per application.
+service ShuffleManager {
+  rpc reportShuffleFetchFailure (ReportShuffleFetchFailureRequest) returns (ReportShuffleFetchFailureResponse);
+}
+
+message ReportShuffleFetchFailureRequest {
+  // appId normally should be omitted, it's used to avoid wrongly request issued from remaining executors of another
+  // app which accidentally has the same shuffle manager port with this app.
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 stageAttemptId = 3;
+  int32 partitionId = 4;
+  string exception = 5;
+  string serverId = 6;
```

Comment:
What do the serverId mean?

Reply:
We already have a proto type called ShuffleServerId. Should we reuse it?

Reply:
It's difficult for us to change old protocol field. If it won't bring too much cost, I prefer that we have a unify style.